static/sass/components/_intro.scss

```diff
@@ -0,0 +1,64 @@
+// Intro Blocks
+// ============
+// Two column (on wide screens) intro and aside
+// Can be seen on case studies
+
+// wrapping container for intro
+.intro-block {
+  @include grid;
```

Comment:
well, I know we are supposed to do the @include statements first (I think the sasslint would yell at me) so that is why it is there. But I didn't even look how it compiles. I know the mixin is wrapped in an @supports statement already.